### PR TITLE
CommandCode: update list of available commands on MAV_CMD enum

### PR DESCRIPTION
**Pull request by TSC21**
I used cog to get all the available commands from `MAV_CMD` enum, excluding:
1. An exclusion list that includes 'SPATIAL', 'USER', 'WAYPOINT' user defined commands;
2. Deprecated commands (by description inspection).
This way, updating the command codes to be used by services.

### CODE REVIEW

File: mavros_msgs/msg/CommandCode.msg [hunk 1]
```diff
@@ -1,37 +1,206 @@
...
+#     enum = sorted(common.enums[ename].items())
+#     enum.pop() # remove ENUM_END
+#
+#     enumt = []
+#     # exception list of commands to not include
+#     exlist = ['SPATIAL', 'USER', 'WAYPOINT']
+#     for k, e in enum:
+#         enumt.extend(e.name[len(ename) + 1:].split('_')[0:1])
+#
+#     enumt = sorted(list(OrderedDict.fromkeys(enumt)))
```
Comment (vooon) [inline mavros_msgs/msg/CommandCode.msg]: What this do?!

Comment (TSC21) [reply mavros_msgs/msg/CommandCode.msg]: It removes the duplicates keys from `enumt` and then sorts them alphabetically.

**Comment by TSC21** [comment]
> Result looks ok, but remove `CMD_` prefix. That may break scripts.

Where do you see the `CMD_` prefix on the enums?

**Comment by vooon** [comment]
Use `set()`!

пн, 9 апр. 2018 г., 14:54 Nuno Marques <notifications@github.com>:

**Comment by vooon** [comment]
In older code: CMD_DO_

пн, 9 апр. 2018 г., 15:02 Nuno Marques <notifications@github.com>:

**Comment by TSC21** [comment]
> Use `set()`!

Done

**Pull request merged**